projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/my-company/my-company-form.ts

```diff
@@ -169,10 +192,29 @@ function verifyDetails(config: MyCompanyConfig, valueKey: string) {
         'contain.text',
         rowConfig[valueKey] || label
       );
-      // TODO: Check property links
-      // if (rowConfig.link) {
-      //   cy.get('div.property a').should('contain.html', rowConfig.link);
-      // }
+
+      const link = getLink(rowConfig);
+      if (link) {
+        // spike todo get context from variables
```

Comment:
```suggestion
        // TODO: spike todo get context from variables
```

Not quite sure what this means so adding the TODO label to check it later.